tests/lib/spread-funcs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Define MATCH when invoked outside of spread context.
```

Comment:
Do we need to keep this function in sync with the spread implementation? If so we should probably add a comment here about it (I guess we don't have to keep it in sync its just nice if it is in sync?)

Reply:
No, it is updated on each run via spread itself